banking/statements/osuuspankki/__init__.py

```diff
@@ -17,7 +17,8 @@
    "VUOKRA": "PAYMENT",
    "VERONPALAUTUS": "DIRECTDEP",
    "PALKKA": "DIRECTDEP",
-   "ARVOPAPERI": "OTHER"
+   "ARVOPAPERI": "OTHER",
+   "LUOTON MAKSU": "REPEATPMT"

```

Comment:
Are you certain that mapping of "LUOTON MAKSU" to "REPEATPMT" is always correct, or the most common case? Do you have any documentation suggesting that?